LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentAuthRequestParams.kt

```diff
@@ -10,7 +9,6 @@ import com.braintreepayments.api.BrowserSwitchOptions
  * @property approvalUrl used for payment approval.
  * @property paymentId of the local payment after creation.
  */
-@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Is it possible to restrict individual params to library scope? It would be good to keep as much internal as possible to avoid future breaking changes. I think `browserSwitchOptions` does not need to be accessible if it can be internal.

Reply:
@sarahkoop restricted the property `browserSwitchOptions`.